package.json

```diff
@@ -1,41 +1,44 @@
 {
     "name": "kettle",
     "description": "Declarative IoC-based framework for HTTP and WebSockets servers on node.js based on express and ws",
-    "version": "1.7.1",
+    "version": "1.7.2",
     "author": {
         "name": "The Fluid Project"
     },
     "bugs": {
         "url": "http://issues.fluidproject.org/browse/KETTLE"
     },
     "scripts": {
-        "test": "node tests/all-tests.js"
+        "pretest": "node node_modules/rimraf/rimraf coverage/* reports/*",
+        "test": "node node_modules/istanbul/lib/cli.js cover tests/all-tests.js --print summary"
```

Comment:
Istanbul is largely deprecated, this seems a reasonable time to upgrade unless you are relying heavily on istanbul-isms.

Reply:
(to nyc, I mean)

Reply:
Also might be a good time to move the report to its own "reports" subdirectory.

Reply:
For an example config and command, see [the pull to update gpii-express to use nyc](https://github.com/GPII/gpii-express/pull/39)